pepper-apis/dsm-core/src/test/java/org/broadinstitute/dsm/service/onchistory/OncHistoryUploadServiceTest.java

```diff
@@ -377,6 +408,31 @@ private static void createFieldSettings() {
         fieldSettingsDao.create(fieldSettings);
     }
 
+    // TODO: temporary until these are added via Liquibase
+    private static void createRequestStatusFieldSettings() {
+        FieldSettingsDao fieldSettingsDao = FieldSettingsDao.of();
+        FieldSettingsDto.Builder builder = new FieldSettingsDto.Builder(instanceDto.getDdpInstanceId());
+        FieldSettingsDto fieldSettings = builder.withFieldType("VOCABULARY")
+                .withColumnName("REQUEST_STATUS")
+                .withDisplayType("OPTIONS")
+                .withPossibleValues("[{\"value\": \"Needs Review\"},"
+                        + "{\"value\": \"Don't Request\"},"
+                        + "{\"value\": \"On Hold\"},"
+                        + "{\"value\": \"Request\"},"
+                        + "{\"value\": \"Sent\"},"
+                        + "{\"value\": \"Received\"},"
+                        + "{\"value\": \"Returned\"},"
+                        + "{\"value\": \"Unable to Obtain\"}]").build();
+        fieldSettingsDao.create(fieldSettings);
```

Comment:
Do these need to be cleaned up after the class is finished?

Reply:
Now that these are deployed as part of Liquibase (PR merged earlier) I can remove this method.